docs/docs/metrics/prometheus/index.html

```diff
@@ -336,54 +336,6 @@ <h2 id="prometheus">Prometheus</h2>
 </code></pre></div><ul>
 <li>The <code>host.docker.internal</code> tells a Prometheus instance running inside a docker container to connect to the host&rsquo;s <code>8081</code> port where Raper runs from a JAR.</li>
 </ul>
-<h3 id="metric-relabelling">Metric Relabelling</h3>
```

Comment:
I'm okay with removing documentation, especially when we're automating the actions elsewhere.

However, I'm afraid I'd need to ask you for a few things.

First, it'd be nice if you make the automatic re-labeling cover all metrics, or at least the ones mentioned in the document. The `repairProgress`is one (and perhaps only one) that I'm missing.

Second, we have a bit obscure setup with how the docs are generated. The HTML file here is just a render. Its source lives in [src/docs/content/docs/metrics/prometheus.md](https://github.com/thelastpickle/cassandra-reaper/blob/master/src/docs/content/docs/metrics/prometheus.md). Can you please change that too?

Reply:
Sure.